tests/drivers/flash/nrf_qspi_nor/src/main.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <drivers/flash.h>
+
+#if (CONFIG_NORDIC_QSPI_NOR - 0)
```

Comment:
```suggestion
#if (CONFIG_NORDIC_QSPI_NOR - 0)
#define FLASH_DEVICE DT_LABEL(DT_INST(0, nordic_qspi_nor))
#else

/* SoC emebded NVM */
#define FLASH_DEVICE DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_LABEL

#ifdef CONFIG_TRUSTED_EXECUTION_NONSECURE
#define FLASH_TEST_REGION_OFFSET FLASH_AREA_OFFSET(image_1_nonsecure)
#else
#define FLASH_TEST_REGION_OFFSET FLASH_AREA_OFFSET(image_1)
#endif

#endif
```

Reply:
for qspi nordic testing area should be somewhere between 0 and DT_REG_SIZE(DT_INST(0, nordic_qspi_nor))

Reply:
Yes, but not totally. In this case it's not about `DT_REG_SIZE` - it's about `size` property. Correct me, if I'm accessing it wrong:
`#define TEST_AREA_MAX DT_PROP(DT_INST(0, nordic_qspi_nor), size)`.
I don't see equivalent of `DT_PROP(node_id, size)`.

Reply:
You are right - I miss that size of non-embedded memory is declared differently than embedded one.
syntax `DT_PROP(DT_INST(0, nordic_qspi_nor), size).` looks correctly to me.